src/midgard/openlr.cc

```diff
@@ -0,0 +1,97 @@
+#include <cmath>
+#include <string>
```

Comment:
We should move this to the baldr namespace since it's not just geometry stuff but really a path on the road network with road network attribution etc.

Reply:
Done